### DISCUSSION TITLE
Limit Selections on FieldType

### DISCUSSION THREAD

**Discussion by Emkaytoo** [solspace/craft-freeform#32] [Feature Requests]
I would find it useful to be able to limit what an author can select from when using the Freeform Form fieldType. I have thought about it in two ways, both or either of which could be useful:
1. Similar to how Entries limit sources, allow for all forms or certain forms to be select while creating the field. This would be useful when I have multiple forms, but only want to allow "Customer Service" and "Technical Support" forms to be selected on a page.
1. Form Groups, and limiting on the basis of those groups. When creating a field, I can select "Customer Service" form group, while on a content-marketing landing page I can select "Landing Page Forms". This would also allow for greater organization for companies with many variations of the same form, for instance when integrating with a marketing automation platform.

Just my ideas.

**Comment by kjmartens** [comment]
Thanks for your feedback, it's been noted. :)

**Comment by kjmartens** [comment]
This one has been a long time coming, but Freeform 5.9+ can now limit the Forms element field type by specific forms. 🙂
